modules/nf-core/seqkit/sort/tests/main.nf.test

```diff
@@ -0,0 +1,59 @@
+nextflow_process {
```

Comment:
Can we have a test for a zipped fastq file please

Reply:
Yes, I'll add more tests.